ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/TransactionService.scala

```diff
@@ -650,10 +712,11 @@ class TransactionService(session: LedgerSession) extends LedgerTestSuite(session
       "TXRejectMultiActorMissingAuth",
       "Reject exercising a multi-actor choice with missing authorizers") { context =>
       for {
-        ledger <- context.participant()
-        Vector(operator, receiver, giver) <- ledger.allocateParties(3)
-        triProposal <- ledger.create(operator, TriProposal(operator, receiver, giver))
-        failure <- ledger.exercise(giver, triProposal.exerciseTriProposalAccept).failed
+        Vector(alpha, beta) <- context.participants(2)
+        Vector(operator, receiver) <- alpha.allocateParties(2)
+        giver <- beta.allocateParty()
+        triProposal <- alpha.create(operator, TriProposal(operator, receiver, giver))
+        failure <- beta.exercise(giver, triProposal.exerciseTriProposalAccept).failed
```

Comment:
Potential flakiness between `alpha` and `beta`.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/2900/commits/0c9203e60b21cdf3ea34816ba143f4790daf6b15